### PR TITLE
chore: Enable lint for TrackTile

### PR DESCRIPTION
**Pull request by bruce-glazier**
## Changes

  - Superficial changes to satisfy the linter

### CODE REVIEW

File: src/components/TrackTile/TrackerDetails/TrackerDetails.tsx [hunk 1]
```diff
@@ -109,62 +109,67 @@ export const TrackerDetails: FC<TrackerDetailsProps> = (props) => {
...
-        const updates: TrackerValue[] = [];
-        const deletes: TrackerValue[] = [];
-        let trackerValueIndex = 0;
-        let delta = convertToStoreUnit(newValue, tracker) - currentValue;
-        while (delta !== 0) {
-          const trackerValue = values[trackerValueIndex];
-          const resourceValue = (trackerValue?.value ?? 0) + delta;
+  const saveNewValue = useMemo(
+    () =>
+      debounce(async (newValue: number) => {
```
Comment (bruce-glazier) [inline src/components/TrackTile/TrackerDetails/TrackerDetails.tsx:114]: See [here](https://github.com/facebook/react/issues/19240) for the reason I made this change. Manual testing demonstrated that everything was incrementing/decrementing correctly after making this change

**Review by namick — DISMISSED** [review]
Nice!

**Review by sternetj — APPROVED** [review]
Nice! Thanks for doing this @bruce-glazier!

File: src/components/TrackTile/ManageTrackers/ManageTrackers.tsx [hunk 2]
```diff
@@ -22,7 +22,8 @@ export type ManageTrackersProps = {
 };
 
 export const ManageTrackers: FC<ManageTrackersProps> = (props) => {
-  const { onOpenTracker, trackerRequestMeta = useTrackers(), icons } = props;
+  const trackerRequestMetaHook = useTrackers();
```
[the remark this answers is not in the document]
Comment (bruce-glazier) [reply src/components/TrackTile/ManageTrackers/ManageTrackers.tsx:25]: Indeed the ESLint rule did not allow the conditional use of hooks but I have an idea for how to preserve the original logic and the lint rule. I will do that in a follow up.

**Pull request merged**